### PR TITLE
New Maints Room: Wikipedia Server Room

### PR DESCRIPTION
**Pull request by YakiAttaki**
## About The Pull Request
This adds a new maints room: the wikipedia server room. Mostly just a trap room that locks someone in and plays a sound for them.
the sound that plays is this: https://youtu.be/nOPIu7isD3s

## Why It's Good For The Game
funny maints traps

## Changelog
:cl:
add: Added a new maints room: the Wikipedia server room
soundadd: added a new sound: wikipedia.ogg
/:cl:

### CODE REVIEW

File: code/game/objects/effects/mainttraps.dm [hunk 1]
```diff
@@ -126,6 +126,35 @@
 	inuse = FALSE
 	return TRUE
 
+/obj/effect/trap/nexus/doorbolt/wikipedia
```
Comment (zeskorion) [inline code/game/objects/effects/mainttraps.dm:129]: if this is a subtype, why are you copying all the code? just make a separate trap that plays sounds

**Review by zeskorion — CHANGES_REQUESTED** [review]
post sound please

File: code/game/objects/effects/mainttraps.dm (hunk 1 above)
Comment (YakiAttaki) [reply code/game/objects/effects/mainttraps.dm:129]: i am stupid is the only answer

**Comment by february-ops** [comment]
Tbh you know i think cbt wikipedia is against rules and that shit ngl

**Comment by Moccha-Bee** [comment]
Just speed merge it already its the greatest maint room in existance

**Comment by YakiAttaki** [comment]
This Is The Greatest Random Maints Room Of All Time

tenor (7)

**Comment by park66665** [comment]
I didn't expect this, but, uh, okay

**Comment by ike709** [comment]
Expected a Half-Life VR reference, got cringe.

**Comment by That0nePerson** [comment]
Also expected a HLVRAI reference, got overused cbt

**Pull request closed without merging**